app/src/main/java/org/fossasia/openevent/general/attendees/AttendeeFragment.kt

```diff
@@ -248,6 +248,12 @@ class AttendeeFragment : Fragment() {
                 rootView.qty.text = " — $it items"
             })
 
+            attendeeFragmentViewModel.totalAmount.observe(this, Observer {
+                if (it != null && it <= 0) {
+                    rootView.country.visibility = View.GONE
+                }
+            })
```

Comment:
ViewModel is there for logic, I don't get why every now and then logic is leaked to view

Imagine Activity and Fragments only contain XML and not Java, and solve problems using that idea

Reply:
resolved